### ISSUE TITLE
"Pick a font" and "Select color" windows are not on Crowdin

### ISSUE THREAD

**Issue by dr03ramos** [obsproject/obs-studio#3598]
## Platform

Operating system and version: Windows 10
OBS Studio version: OBS 26.0.2 (64-bit)

## Expected Behavior

Translators should be able to translate that screens on Crowdin.
![image](https://user-images.githubusercontent.com/50461210/95690727-16f2ce80-0bf0-11eb-9881-c3543a3b455f.png)
![image](https://user-images.githubusercontent.com/50461210/95690858-1dce1100-0bf1-11eb-9f43-9d610631c26e.png)

## Current Behavior

The texts are not on Crowdin.

## Steps to Reproduce

1. Open OBS and add a text source
2. Click "Select font"
3. Take any string (e.g. "Writing System") and search it on [OBS project on Crowdin](https://crowdin.com/project/obs-studio)
4. Note that the string is missing and isn't translatable

**Comment by WizardCM** [comment]
I noticed this earlier in the year.

We have two potential directions to go:

1. We create our own translation keys in our standard .ini files. This may require some tweaking (to strip out symbols & spaces in translation keys, for example).
2. We include the standard Qt translation files as a fallback, and use the standard QTranslator capability. I don't know what the process would be to add these translation files to OBS.

I've been able to make the first option functional, but am not a fan of potentially creating many, many translation keys for something that could unexpectedly change in a Qt update. https://github.com/WizardCM/obs-studio/commit/7679e52e1b8ebfb484a2dd33eef9220f8a1e8716

**Comment by WizardCM** [comment]
After a quick test, looks like while https://github.com/obsproject/obs-studio/commit/c4840ddba05a8c2d01568445645097704595ea47 does set the Qt locale which would affect these dialogs, we don't ship Qt translation files so the text continues to fall back to English.

**Comment by Fenrirthviti** [comment]
This is not a bug, but a feature request. PRs are welcome.
